modules/mono/build_scripts/make_android_mono_config.py

```diff
@@ -27,7 +27,6 @@ def generate_compressed_config(config_src, output_dir):
 
 
 namespace {
-
```

Comment:
No empty new line allowed after namespaces either?

Reply:
I have no strong opinion about it, but if we want to keep allowing them, I guess @aaronfranke's script could be improved to leave out cases starting with `^namespace {`.

Reply:
In order for `sed` to modify newlines, we have to use the `-z` option, which treats the whole file as a single line, and newline characters as normal characters (technically lines are delimited by `null`). So it would be tricky to make a script that excludes `namespace`. One easy idea is to just use the script as-is and manually revert changes to `namespace` blocks.

Reply:
`sed` is indeed too limited for multiple line matching, but `perl -pi -e` works well: https://github.com/godotengine/godot/pull/38738/commits/07bc4e2f96f8f47991339654ff4ab16acc19d44f